Troubleshooting for NGINX error log: Connection Refused (#4930)
mattermost/docs: source/install/config-proxy-nginx.rst

SSL and HTTP/2 with server push are enabled in the provided configuration example.

.. note::
  
  - If you're going to use Let's Encrypt to manage your SSL certificate, stop here at step 3 here and see the `NGINX HTTP/2 & SSL product documentation <https://docs.mattermost.com/install/config-ssl-http2-nginx.html>`__ for details.
  - You'll need valid SSL certificates in order for NGINX to pin the certificates properly. Additionally, your browser must have permissions to accept the certificate as a valid CA-signed certificate.

>   - Examples in this documentation are using a private IP address, which may not match your network configuration. 

Reviewer: ```suggestion
  - Note that the IP address included in the examples in this documentation may not match your network configuration.
```

      - "80"
      - "443"

> Why does NGINX fail when installing Gitlab CE with Mattermost on Azure?

Reviewer: ```suggestion
Why does NGINX fail when installing GitLab CE with Mattermost on Azure?
```
